### ISSUE TITLE
Initial learning rate for detection is set as 0.0002, which is inconsistent to the value 0.0001 in paper 

### ISSUE THREAD

**Issue by JierunChen** [snap-research/EfficientFormer#18]
Hi, could you please clarify the initial learning rate for detection? It's a key hyper-parameter that can affect the performance after training for 12 epochs.
